fiftyone/core/annotation.py

```
@@ -78,3 +81,107 @@
+
+        if f.name == "bounding_box" and field.document_type == fol.Detection:
+            # bounding_box is a list of floats field, but really a 4-tuple of
+            # [0, 1] floats, omit for special handling by the App
+            continue
+
+        try:
+            attributes[f.name] = compute_annotation_schema(
+                collection, f"{field_name}.{f.name}"
+            )
+        except:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)